integration_tests/pages/apply/risks-and-needs/risk-of-serious-harm/roshSummaryPage.ts

```diff
@@ -22,4 +24,32 @@ export default class RoshSummaryPage extends ApplyPage {
       }),
     )
   }
+
+  shouldShowRiskData = (risks: RoshRisks, dateOfOasysImport: string): void => {
+    cy.get('p').contains(
+      `Imported from OASys on ${DateFormats.isoDateToUIDate(dateOfOasysImport, {
+        format: 'medium',
+      })}, last updated on ${DateFormats.isoDateToUIDate(risks.lastUpdated, {
+        format: 'medium',
+      })}`,
+    )
+
+    cy.get('h3').contains(`${risks.overallRisk.toLocaleUpperCase()} RoSH`)
+
+    cy.get('.rosh-widget__table').within($row => {
+      cy.wrap($row).get('th').contains('Children').get('td').contains(risks.riskToChildren, { matchCase: false })
```

Comment:
Is this `$row` variable name using the `$` to denote the fact that it's been assigned a DOM element?

Reply:
it used to be quite a common convention (possibly still is?) and I do think it's helpful. It believe it goes back to the JQuery alias of `$(selector)` and would signal that the variable had a JQuery object assigned. That history may or may not make adopting the convention attractive?

Reply:
Sounds helpful to me! I'm sold 🙂